databuilder/extractor/dashboard/tableau/tableau_dashboard_constants.py

```diff
@@ -0,0 +1,5 @@
+API_VERSION = 'api_version'
+TABLEAU_HOST = 'tableau_host'
+SITE_NAME = 'site_name'
+TABLEAU_ACCESS_TOKEN_NAME = 'tableau_personal_access_token_name'
+TABLEAU_ACCESS_TOKEN_SECRET = 'tableau_personal_access_token_secret'
```

Comment:
Are there other token types than personal? Just curious about how best to name this config.